app/models/job_invocation.rb

```diff
@@ -15,7 +15,8 @@ class JobInvocation < ActiveRecord::Base
   validates :job_name, :presence => true
   validates_associated :targeting, :template_invocations
 
-  scoped_search :on => :job_name
+  scoped_search :on => :job_name, :complete_value => true
+  scoped_search :on => :description, :complete_value => true
```

Comment:
I'd say it's generic scoped search issue and searching can still still useful, 3/4 templates we ship do not contain quotes. @stbenjam would you mind opening scoped_search and a tracking issue and linking it to #13147 for this? I think it does not block this PR to be merged right?